wemake_python_styleguide/visitors/ast/operators.py

```diff
@@ -60,6 +59,7 @@ class UselessOperatorsVisitor(base.BaseNodeVisitor):
             ast.Mult,
             ast.Pow,
             ast.Mod,
+            ast.BitXor,
```

Comment:
What about `|` operator and `0`?

Reply:
```
>>> 10 | 0
10
>>> 101 | 0
101
```

Reply:
Oh yeah, I'll add it.

On a side note, why is xor-ing with 1 a useless operation?

```
>>> 3 ^ 1
2
```

I thought at first you meant `~1` but that would change the value as well.

```
>>> 3 ^ ~1
-3
```

Reply:
This is my mistake. `^` is useless with `0`:

```
>>> 3 ^ 0
3
>>> 10 ^ 0
10
```

Reply:
@sobolevn Quick question: using `&` with `0` is also not useless, right?